wrangler-transform/src/e2e-test/features/Wrangler/DataTypeParsers.feature

```diff
@@ -24,8 +24,6 @@ Feature:  datatype parsers
     Then Replace input plugin property: "project" with value: "projectId"
     Then Replace input plugin property: "dataset" with value: "dataset"
     Then Replace input plugin property: "table" with value: "bqSourceTable"
-    Then Click on the Get Schema button
```

Comment:
why is this removed?

Reply:
As the pipelines are exported.In almost all the PR's the issue is when we try to get schema then randomly in most of the fields it is going null when we click on get schema and validate button. And it also changes the schema . So just to solve that issue we removed that part from here.

Reply:
What do you mean by this?
```
when we try to get schema then randomly in most of the fields it is going null when we click on get schema and validate button. And it also changes the schema
```

Can you please elaborate with the help of screenshots?

Reply:
added the schema and validation step.